tmt/__init__.py

```diff
@@ -1,8 +1,8 @@
 """ Test Management Tool """
 
-# starting with setuptools_scm 7.0 it's possible to import __version__ directly
-# to support setuptools_scm 6.0 (in EL9), we can only import version
-from tmt._version import version as __version__  # noqa: F401
+import importlib.metadata
+
+__version__ = importlib.metadata.version(__name__)
```

Comment:
Hm, so how exactly is now the version constructed? I tried checking it on this branch and I get this:

```
(tmt) psss@fedora:~/git/tmt (remove_version_file)> tmt --version
tmt version: 1.27.1.dev19+g56353641.d20230914
```

The commit `56353641` does not exist in the git repo anymore and it's 25 commits since the last release:

```
> hatch version
1.28.0.dev25+g3c27e874
```

Any idea what's wrong here?

Reply:
@psss it's getting it from the built (and installed) packages metadata. `hatch version` is giving you the current scm-based version. If you do a pip install they will match.

Reply:
Ok, I thought it would work with `pip install --editable` and rather report the current commit. But most important is that it works nicely within the package (which seems to be ok).

Reply:
Editable would mean feet shooting :)
https://github.com/maresb/hatch-vcs-footgun-example

Reply:
Ok, let's shoot on our feet later ;-)